rskj-core/src/main/resources/reference.conf

```diff
@@ -405,7 +405,7 @@ cache {
     # Processing 134 blocks grows the commitedCache to 100K entries, or approximately
     # or 25 Mbytes. A cache of about 100 Mbytes seems rasonable. Anyway, for
     # precaution we'll set the limit to 100K entries.
-    max-elements: 100000
+    max-elements: 1000000
```

Comment:
we should update the comments preceding this value, right?

Reply:
Done!